test_cases/security/security_test_runner.sh

```
@@ -0,0 +1,108 @@
+echo -e "${BLUE}=== TASKER SECURITY TEST RUNNER ===${NC}"
+echo -e "${BLUE}Testing negative security scenarios${NC}"
+echo -e "${BLUE}Expected behavior: ALL tests should FAIL${NC}"
+echo ""
+
+# Set PATH for mock commands and skip host validation
+export PATH="../test_scripts:$PATH"
+
+# Function to test a single security test file
+test_security_file() {
+    local test_file="$1"
+    local test_name=$(basename "$test_file" .txt)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)